plugin/src/main/java/com/google/errorprone/xplat/checker/AllowLegacyTime.java

```diff
@@ -0,0 +1,18 @@
+package com.google.errorprone.xplat.checker;
+
+import static java.lang.annotation.ElementType.CONSTRUCTOR;
+import static java.lang.annotation.ElementType.METHOD;
+import static java.lang.annotation.ElementType.PARAMETER;
+import static java.lang.annotation.ElementType.FIELD;
+import static java.lang.annotation.ElementType.LOCAL_VARIABLE;
+
+import java.lang.annotation.Target;
+
+/**
+ * Annotation that indicates the explicit use of java.util.Calendar, in order to override the
+ * CalendarClassBan xplat error prone checker.
+ */
+@Target({CONSTRUCTOR, METHOD, PARAMETER, FIELD, LOCAL_VARIABLE})
```

Comment:
Add `@Retention(RetentionPolicy.SOURCE)`  on the interface so that the annotation does not go into the class file (which is the default): https://docs.oracle.com/javase/7/docs/api/java/lang/annotation/RetentionPolicy.html